### DISCUSSION TITLE
Desde 🇲🇽 México y 🇧🇴 Bolivia para el mundo 🌏

### DISCUSSION THREAD

**Discussion by CarlaMamaniChavez** [gittogethers/octogatosconf22#51] [General]
Somos 🦸🏻‍♂️Luis y 🦸🏻‍♀️Carly, nos encanta compartir conocimientos 🎁 a toda la comunidad tecnológica 👨🏻‍💻👩🏻‍💻 hispano hablante. Si quieres saber mas sobre nosotros nos puedes encontrar como:  @CarlaMamaniChavez y @icebeam7  🤩

Estamos muy emocionados de poder estar presentes en 🥳 #octogatos2022 donde hablaremos sobre Inteligencia Artificial Responsable 🤖 😎

Prometemos dar todo lo mejor en esta sesión 🤩🤯.

**Comment by AndreaGriffiths11** [comment]
¡Bienvenidos!!! Esta charla me tiene superemocionada
